### DISCUSSION TITLE
ConnectBot sends same keycode for Ctrl-Arrow as for Arrow

### DISCUSSION THREAD

**Discussion by ecm-pushbx** [dosemu2/dosemu2#1746] [General]
Just to notify you all that I reported this bug to ConnectBot, as it appears not to be a dosemu2 bug: https://github.com/connectbot/connectbot/issues/1146

**Comment by stsp** [comment]
I don't know if anyone here knows
or is interested in what connectbot is,
so I would guess its not too informative
to post that here.
But I can only speak for myself of course.
Open a socat bug and for sure most of
dosemu2 ppl would appreciate. :)

**Comment by ecm-pushbx** [comment]
I don't think socat has a bug tracker, unfortunately. The page at http://www.dest-unreach.org/socat/ just says:

> ## Contact ##
> If you have more questions, please contact [socat@dest-unreach.org](mailto:socat@dest-unreach.org)
